### PR TITLE
C51-377: Fix case card activity count popup location

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
In the Case List, the Activities Count Popup sometimes does not get fully displayed.

## Before
2019-03-19 at 3 21 PM

## After
2019-03-19 at 3 21 PM

## Technical Discussion
1. Similar to https://github.com/compucorp/uk.co.compucorp.civicase/pull/188

Previously inside `case-activity-count.directive.html`, the `uib-popover` was used. But the problem with this directive, is that it needs to be attached to its parent of to the Body. But not any other elements.

If we attach it to its parent(it was like that before), then the popover will be cut by the Scrollbar area.
If attached to the body, the styles wont work because body is outside of #bootstrap-theme.

That is why, `civicase-popover` directive has been used.
This has some drawbacks,
For example, when scrolling the Case List, the popover does not scroll with it.

So this is not a perfect solution, but given the timeline, I have prioritised showing the popover without being cut.

2. The `z-index` of `.civicase__case-activity-count__popover ` has been changed to `11`, so that the popover appears over the `case list pager`, which also has a `z-index` of `10`. 

3. In the new implementation the popup is shown in the bottom direction, so the `.arrow` border direction has been changed from `top` to `bottom`.

4. `margin-left` property has been removed from `.civicase__case-activity-count__popover`. It was added in https://github.com/compucorp/uk.co.compucorp.civicase/pull/188 to fix the position of the arrow, but its not required anymore.

### CODE REVIEW

File: scss/components/_case-activity-count.scss [hunk 1]
```diff
@@ -2,13 +2,12 @@
   border: 1px solid $gray-light;
   border-radius: 2px;
   box-shadow: $box-shadow-tooltip;
-  margin-left: 10px;
   padding: 0;
   white-space: nowrap;
-  z-index: 5;
+  z-index: 10;
```
Comment (swastikpareek) [inline scss/components/_case-activity-count.scss]: What is the need of changing the z-index?

Comment (deb1990) [reply scss/components/_case-activity-count.scss]: This way the popup comes above of the case list pager.

Comment (swastikpareek) [reply scss/components/_case-activity-count.scss]: I agree, but few points.
Why z-index: 5 was not working 
and If it tends to fix something. Should we add it in the PR description ?

Comment (deb1990) [reply scss/components/_case-activity-count.scss]: Added to the PR description

**Pull request merged**